src/gui/speedlimitdialog.cpp

```diff
@@ -57,9 +57,9 @@ SpeedLimitDialog::SpeedLimitDialog(QWidget *parent)
     m_ui->setupUi(this);
 
     m_ui->labelGlobalSpeedIcon->setPixmap(Utils::Gui::scaledPixmapSvg(UIThemeManager::instance()->getIconPath(QLatin1String("slow_off"))
-                                                , this, style()->pixelMetric(QStyle::PM_SmallIconSize, nullptr, this)));
```

Comment:
use `QStyle::PM_LargeIconSize`

Reply:
From theming POV, is it worth to use `style()->pixelMetric()`?

Reply:
it's much more useful for Linux peeps, where OS implements QStyle. Also, Qt recommends using QStyle wherever possible.

Reply:
Updated.